cscs-checks/prgenv/cpu_target_check.py

```diff
@@ -9,15 +9,16 @@
 
 @rfm.simple_test
 class CrayCPUTargetTest(rfm.RunOnlyRegressionTest):
-    def __init__(self):
-        self.descr = 'Checks whether CRAY_CPU_TARGET is set'
-        self.valid_systems = ['daint:login', 'dom:login']
-        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
-                                    'PrgEnv-intel', 'PrgEnv-pgi']
-        self.sourcesdir = None
-        self.executable = 'echo CRAY_CPU_TARGET=$CRAY_CPU_TARGET'
+    descr = 'Checks whether CRAY_CPU_TARGET is set'
+    valid_systems = ['daint:login', 'dom:login']
+    valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu', 'PrgEnv-intel',
+                           'PrgEnv-pgi', 'PrgEnv-nvidia']
```

Comment:
`PrgEnv-nvidia` is missing in https://github.com/eth-cscs/reframe/blob/master/config/cscs.py#L339

Reply:
Can you open a PR to fix it?

Reply:
https://github.com/eth-cscs/reframe/pull/2025

Reply:
#2025 has been merged